@here/harp-atlas-tools/src/cli-atlas-generator.ts

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env node
```

Comment:
This doesn't pass licence check in current form @sschuberth, @haraldF. 
Is this ok, to have exception for scripts to not start strictly with licence ?

Reply:
I'm no longer working @heremaps. Please contact @tsteenbe.

Reply:
License check should be ok, it does not allow line space between shebang and license header.